### PR TITLE
fix: a few bugs on details

### PR DESCRIPTION
**Pull request by JackShort**
* hover listing state
* tweet url

### CODE REVIEW

File: src/nft/components/details/AssetPriceDetails.tsx [hunk 1]
```diff
@@ -96,6 +96,7 @@ const MarketplaceIcon = styled.img`
 `
```
Comment (cbachmeier) [inline src/nft/components/details/AssetPriceDetails.tsx:96]: While you're in the area, could you hide `Seller` if owner data is undefined?

File: src/nft/components/details/AssetPriceDetails.tsx [hunk 2]
```diff
@@ -96,6 +96,7 @@ const MarketplaceIcon = styled.img`
...
 
 const BuyNowButton = styled.div<{ assetInBag: boolean; margin: boolean; useAccentColor: boolean }>`
+  position: relative;
```
Comment (cbachmeier) [inline src/nft/components/details/AssetPriceDetails.tsx:99]: not getting the new hover state

Comment (JackShort) [reply src/nft/components/details/AssetPriceDetails.tsx:99]: I am getting it on all browsers - it should be really subtle

File: src/nft/components/details/AssetPriceDetails.tsx [hunk 3]
```diff
@@ -393,9 +394,11 @@ export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps)
       )}
       {isForSale && (
         <OwnerInformationContainer>
-          <ThemedText.BodySmall color="textSecondary" lineHeight="20px">
-            Seller:
-          </ThemedText.BodySmall>
+          {asset.owner.address && (
```
Comment (cbachmeier) [inline src/nft/components/details/AssetPriceDetails.tsx:397]: might need to cast as a bool, still seeing it here `nfts/asset/0xded56b58644373020393ad35b9eb5841296b886b/1`

File: src/nft/components/details/AssetPriceDetails.tsx (hunk 2 above)
Comment (cbachmeier) [reply src/nft/components/details/AssetPriceDetails.tsx:99]: can see it in light mode but personally can't tell on dark, that's on design though

**Review by cbachmeier — APPROVED** [review]
LGTM minor bug with Seller hiding

File: src/nft/components/details/AssetPriceDetails.tsx (hunk 3 above)
Comment (JackShort) [reply src/nft/components/details/AssetPriceDetails.tsx:397]: ohh this was because its an erc1155

**Pull request merged**